src/Ddd/Masa.Contrib.Ddd.Domain.Repository.EF/Repository.cs

```diff
@@ -122,19 +145,33 @@ public override async Task<IEnumerable<TEntity>> GetListAsync(CancellationToken
     }
 
     /// <summary>
-    ///
+    /// Get a paginated list after sorting by condition
     /// </summary>
-    /// <param name="predicate">condition</param>
-    /// <param name="skip"></param>
-    /// <param name="take"></param>
-    /// <param name="sorting">asc or desc, default asc</param>
+    /// <param name="predicate"> A function to test each element for a condition</param>
+    /// <param name="skip">The number of elements to skip before returning the remaining elements</param>
+    /// <param name="take">The number of elements to return</param>
+    /// <param name="sorting">sort parameters</param>
```

Comment:
If sorting is field name, it need a name that doesn't create ambiguity. Because sorting let me think it is `field asc` or `field desc`.

Reply:
processed